jujugui/static/gui/src/app/app.js

```diff
@@ -752,7 +755,7 @@ YUI.add('juju-gui', function(Y) {
           switchModel={views.utils.switchModel.bind(
             this, this.createSocketURL.bind(this), this.switchEnv.bind(this))}
           user={this._getAuth()}
-          users={this.get('users')}
+          users={Y.clone(this.get('users'), true)}
```

Comment:
Heh we had a chat about this and because this is a complex operation I'd rather not reproduce it poorly in multiple places throughout the code. At the point when we can remove YUI we will be easily able to grep to replace this with a proper alternative.